src/actions/__tests__/cursorDown.ts

```diff
@@ -79,6 +85,33 @@ describe('normal view', () => {
 
     expectPathToEqual(stateNew, stateNew.cursor, ['a', 'm'])
   })
+
+  it('should navigate correctly in descending sorted list with formatted text', () => {
```

Comment:
Avoid using the word "correctly" in a test, as it does not describe *what* the correct behavior is. Instead, describe the expected behavior.

`it('move cursor from formatted thought to first unformatted thought in descending order', ...)`